### PR TITLE
chore: Dropdown options should be enabled  for all users

### PR DESCRIPTION
**Pull request by suyash-naithani**
Signed-off-by: suyash-naithani <suyash.naithani@gmail.com>

This issue fixes [9116](https://issues.redhat.com/browse/MGDSTRM-9116)

### CODE REVIEW

File: src/app/modules/OpenshiftStreams/components/StreamsTable/StreamsTable.tsx [hunk 1]
```diff
@@ -255,6 +255,7 @@ export const StreamsTable: FunctionComponent<StreamsTableProps> = ({
       {
         title: t("view_details"),
         id: "view-instanceDrawerInstance",
+        disabled: false,
```
Comment (ajaypratap003) [inline src/app/modules/OpenshiftStreams/components/StreamsTable/StreamsTable.tsx:258]: we don't need to add disabled property. You should remove additionalProps and tooltipProps where we need to show kebab options for all users.

**Review by ajaypratap003 — CHANGES_REQUESTED** [review]
Follow inline comment.

**Comment by ajaypratap003** [comment]
@suyash-naithani  Please consider below minor changes related to this PR.
1. Make sure every row should show mouse hover pointer for all users. Fir now it's showing for org-admin and owner but not for non-owner.
2. Ever row should show vertical blue line on selected row. see below scheenshot.
image

**Comment by riccardo-forina** [comment]
Good points Ajay! I think it would be ok to do that when moving this code to the ui-components repo to use the shared table there and keep this just as a bug fix for the reported issue. What do you think?

**Review by ajaypratap003 — APPROVED** [review]

**Pull request merged**